doc/dune-files.rst

```diff
@@ -664,20 +664,20 @@ The syntax is as follows:
 ``<section>`` is the installation section, as described in the opam
```

Comment:
Since we are documenting this completely, I think it's best to not mention opam here. Just to keep things as simple as possible. What about:

```
``<section>`` is the installation section which determines where the files will be installed.  Dune supports the following sections:
```

Reply:
But maybe we could mention somewhere in this section that this is modelled and compatible with opam install files.